### PR TITLE
feature/manage-filters-api

### LINKED ISSUE #3567: Add endpoints to manage Filters

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========

Allow the filters to be managed by the API

Fixes #3567

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Ready for review

File: lib/pf/AccessScopes.pm [hunk 1]
```diff
@@ -0,0 +1,177 @@
+package pf::AccessScopes;
```
Comment (julsemaan) [inline lib/pf/AccessScopes.pm]: Add proper perldoc to the subs in this module

File: lib/pf/ConfigStore.pm [hunk 2]
```diff
@@ -82,27 +82,32 @@ sub _buildCachedConfig {
...
+        }
+    );
+}
 
-            my $config = eval {pf::IniFiles->new(%args)};
-            if ($@) {
-                get_logger->error("Error opening $file_path : $@");
-                return undef;
-            }
+sub configIniFile {
```
Comment (julsemaan) [inline lib/pf/ConfigStore.pm]: perldoc

File: lib/pf/pfcmd/checkup.pm [hunk 3]
```diff
@@ -1319,6 +1325,27 @@ sub pfdetect {
     }
 }
 
+=head2 validate_access_filters
+
+validate_access_filters
```
Comment (julsemaan) [inline lib/pf/pfcmd/checkup.pm]: something more descriptive would be nice

File: lib/pfconfig/namespaces/config/Realm.pm [hunk 4]
```diff
@@ -27,12 +27,20 @@ use base 'pfconfig::namespaces::config';
 
 sub init {
     my ($self) = @_;
+<<<<<<< d379751f74e7f1aba0b2270f80537f4b3404c5f4
```
Comment (julsemaan) [inline lib/pfconfig/namespaces/config/Realm.pm]: You have a merge conflict here

File: lib/pf/pfcmd/checkup.pm (hunk 3 above)
Comment (jrouzierinverse) [reply lib/pf/pfcmd/checkup.pm]: how about 'Validate Access Filters'?

**Comment by jrouzierinverse** [comment]
Ready for rereview

**Comment by julsemaan** [comment]
Rebase and we can merge

**Pull request merged**